nodejs-patternfly/nodejs-patternfly.spec

```diff
@@ -2,69 +2,76 @@
 %global enable_tests 0
 
 Name: nodejs-%{npm_name}
-Version: 3.29.13
+Version: 3.32.1
 Release: 1%{?dist}
-Summary: A reference implementation of PatternFly based on Bootstrap v3
+Summary: This reference implementation of PatternFly is based on [Bootstrap v3](http://getbootstrap
```

Comment:
Could you change this back? I'm not sure yet how we should deal with this better in npm2rpm yet.